src/Javis.jl

```diff
@@ -317,6 +329,91 @@ function get_javis_frame(video, objects, frame)
         # if object is in global layer this changes the background settings
         update_background_settings!(background_settings, object)
     end
+end
+
+function get_layer_frame(video, layer, frame)
+    Drawing(layer.width, layer.height, :image)
+    objects = layer.children
+    render_objects(objects, video, frame)
+    if frame in get_frames(layer)
+        # call currently active actions and their transformations for each layer
+        actions = layer.actions
+        if !isempty(actions)
```

Comment:
is this if necessary? I think it shouldn't be as the `for` loop will be just empty